frontend/src/types.ts

```diff
@@ -2606,12 +2606,13 @@ export interface SurveyAppearance {
     thankYouMessageDescriptionContentType?: SurveyQuestionDescriptionContentType
     autoDisappear?: boolean
     position?: string
+    shuffleQuestions?: boolean
+    surveyPopupDelay?: number
```

Comment:
```suggestion
    surveyPopupDelaySeconds?: number
```

Adding the units for popup delay would be great here.